deploy/packages/bbp-packages.yaml

```diff
@@ -46,7 +46,7 @@ packages:
       #- python
     specs:
       - parquet-converters@0.3
-      - placement-algorithm@2.0.0
+      - placement-algorithm@2.0.1%gcc@8.2.0
```

Comment:
I have noticed that with `%gcc@8.2.0` the resulting module works ~30% faster, and that's a speedup that would be nice to have considering that `placement-algorithm` run takes hours for 10M circuit.
Would be interesting to understand the reason for the difference in the performance, but I have investigated it yet.

Reply:
CI is failing because of `%gcc@8.2.0` as well.
Apparently once again I'm doing something different locally. ^^